packages/aws-cdk/lib/init-templates/jsii/typescript/package.template.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "%name%",
+  "version": "0.1.0",
+  "description": "A sample CDK app",
+  "keywords": [
+    "cdk",
+    "sns",
+    "sqs",
+    "jsii"
+  ],
+  "author": {
+    "name": "[YOUR NAME HERE]",
+    "email": "[YOUR EMAIL HERE]",
+    "url": "[YOUR WEBSITE HERE]"
+  },
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "[GIT URL HERE]",
+    "directory": "lib/"
+  },
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "scripts": {
+    "build": "jsii",
+    "watch": "jsii -w",
+    "package": "jsii-pacmak -v",
+    "test": "jest",
```

Comment:
There isn't any `jest` configuration, so it's probably not going to work as nice as it could.